packages/react-component-library/cypress/specs/Autocomplete/index.spec.ts

```diff
@@ -0,0 +1,42 @@
+import { describe, cy, it, before } from 'local-cypress'
```

Comment:
Would be nice to sort these:

```suggestion
import { before, cy, describe, it } from 'local-cypress'
```